src/main/groovy/org/jetbrains/intellij/tasks/RunPluginVerifierTask.groovy

```diff
@@ -107,6 +124,38 @@ class RunPluginVerifierTask extends ConventionTask {
         this.ideVersions = ideVersions
     }
 
+    /**
+     * Returns a list of the paths to locally installed IDE distributions that should be used for verification.
+     *
+     * @return locally installed IDEs list
+     */
+    @Input
+    List<String> getLocalPaths() {
+        return Utils.stringListInput(localPaths)
+    }
+
+    /**
+     * Sets a list of the paths to locally installed IDE distributions that should be used for verification.
+     * Property does not override {@link #ideVersions}.
+     * Accepts list of {@link String} or {@link Closure}.
+     *
+     * @param localPaths list of IDE versions
+     */
+    void setLocalPaths(List<Object> localPaths) {
+        this.localPaths = localPaths
+    }
+
+    /**
+     * Sets a list of the paths to locally installed IDE distributions that should be used for verification.
+     * Property does not override {@link #ideVersions}.
+     * Accepts list of {@link String} or {@link Closure}.
+     *
+     * @param localPaths list of IDE versions
+     */
+    void localPaths(List<Object> localPaths) {
```

Comment:
dup?

Reply:
Nope, the Gradle plugin has two setters for properties - with and without `set` prefix. I simply follow that convention.